Lib.AspNetCore.ServerSentEvents/ServerSentEventsService.cs

```diff
@@ -57,6 +57,11 @@ public ServerSentEventsService(IOptions<ServerSentEventsServiceOptions<ServerSen
                 {
                     ClientDisconnected += (sender, args) => serviceOptions.OnClientDisconnected((IServerSentEventsService)sender, args);
                 }
+
+                if (serviceOptions.ValidationHandler != null) 
```

Comment:
Is that null check needed? If it will be set tu null nothing bad will happen right?

Reply:
True, it was copy-pasted from the event bits above which do need to be different.